STARPORT.md

```diff
@@ -32,6 +36,32 @@ starport network proposal approve [chainID] 1,4,5,6 --nightly
 
 Replace comma-separated values with a list of `proposalID` being accepted. Replace `approve` with `reject` to reject proposals instead.
 
+### Joining as a validator
+
+**IMPORTANT: Be sure to run the following on the machine you'll use for the testnet.**
+
+Run the following command from a server to propose yourself as a validator:
+
+```
+starport network chain join [chainID] --nightly
+```
+
+Follow the prompts to provide information about the validator. Starport will download the source code of the blockchain node, build, initialize and create and send two proposals to SPN: to add an account and to add a validator with self-delegation. By running a `join` command you act as a "validator". When filling out the required parameters ensure to include the **'stake'** word after the required values for the inputs to be accepted. If the terminal gets an error or hangs then you can also try: `starport network chain join [chainID] --nightly --keyring-backend "test"`
```

Comment:
, initialize and create and send two proposals could be changed to:
, initialize, create and send two proposals